React/CalcApp/e2e/src/fixtures/example.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "Using fixtures to represent data",
```

Comment:
is this necessary? we will not deal with e2e so i do not want to create confusion

Reply:
I removed the e2e folder with all its content